changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "3.3.4-beta1": [
+      "[Fixed] Close button in release notes dialog can be clicked - #17438",
+      "[Fixed] Scrolling works as expected in the \"Commit Reachability\" dialog - #17421",
+      "[Fixed] Tip of comment bubbles in Pull Request notifications is rendered correctly - #17411",
+      "[Improved] Upgrade to Electron v26.2.1 - #17408",
+      "[Improved] Add commit message length warning toggle - #17370. Thanks @rystills!",
+      "[Improved] Clarified the outcome of toggling the setting under \"Background Updates\" in the \"Advanced\" settings - #17389",
+      "[Improved] Undo link when resolving conflicts is now a button - #17373"
```

```suggestion
      "[Improved] Upgrade to Electron v26.2.1 - #17408"
```

Already in beta 3.3.3-beta1